Documentation update readme

WernerAssis opened the pull request:
Desenvolvimento da Documentação da interface do ADinfo.

WernerAssis committed:
alteração de npm run dev para npm start

WernerAssis committed:
teste de commit Readme

WernerAssis committed:
primeira versão do readme, testes

WernerAssis committed:
correções do primeiro commit do readme

WernerAssis committed:
correção de textos

Commit:
docs: Adição de novas informações e ajustes esteticos.

Commit:
docs: Descentralização e alteração de um erro no código.

WernerAssis committed:
docs: correção de alinhamento da DOC.

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+<div align="center"> <img src="https://raw.githubusercontent.com/DP6/templates-centro-de-inovacoes/main/public/images/centro_de_inovacao_dp6.png" height="100px" /> </div>
 
-> A Vue.js project
+**ADInfo-interface** é a interface do [penguin ADinfo](https://github.com/DP6/adinfo-interface) ferramenta que foi desenvolvida para facilitar o controle e padronização do uso de parametrização e nomenclatura de mídia digital. Os principais componentes no uso da aplicação são a configuração, um JSON contendo quais campos são aceitos na taxonomia de mídia, o permissionamento para controle de ações ...
```
- Usar "Adinfo" em vez de "ADInfo"
- Usar o link da API, não da interface
- "o permissionamento para controle de ações"
- Retirar o "e": permitidas por nível de acesso, o arquivo de parametrização

WernerAssis replied:
os pontos observados foram corrigidos.

LucasTonetto commented on README.md:
````diff
@@ -1,18 +1,115 @@
-``` bash
-# install dependencies
-npm install
+Nossa interface foi idealizada para **simplificar** e **otimizar** a parametrização de links de forma altamente costumizável, sendo uma ferramenta descomplicada de se utilizar e que irá centralizar de **forma segura** as informações mais relevantes do seu negócio. 
````
- Substituir "Nossa interface" por "A interface"
- O que seria "centralizar de forma segura as informações de negócio"?

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
 
-# serve with hot reload at localhost:8080
-npm run dev
+Por meio de controle de acessos, histórico de campanhas e versionamento de taxonomia, torna a interface do ADinfo muito amigável, visualmente simples e adaptável as necessidades de cada marca.
```
Acho que esse trecho pode retirar

WernerAssis replied:
ok

LucasTonetto commented on README.md:
````diff
@@ -1,18 +1,115 @@
-```
+# Requisitos para utilização
+- Ambiente de hospedagem de aplicações, em nuvem, sugestão :
+( [Heroku](https://www.heroku.com/), [App Engine](https://cloud.google.com/appengine?utm_source=google&utm_medium=cpc&utm_campaign=latam-BR-all-pt-dr-SKWS-all-all-trial-e-dr-1011454-LUAC0008679&utm_content=text-ad-none-any-DEV_c-CRE_429626774325-ADGP_Hybrid%20%7C%20SKWS%20-%20EXA%20%7C%20Txt%20~%20Compute_App-Engine-KWID_43700040369789872-kwd-359982465286&utm_term=KW_app%20engine-ST_App%20Engine& ...
````
- Tirar a vírgula antes do "em nuvem"
- Especificar o serviço da AWS onde poderia ser hospedado
- HostGator não é uma hospedagem em nuvem. Sugiro alterar o texto para "hospedagem" sem especificar o "em nuvem"

WernerAssis replied:
- HostGator removido 
- texto alterado para somente ''- Ambiente de hospedagem de aplicações. sugestão:''

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+# Requisitos para utilização
+- Ambiente de hospedagem de aplicações, em nuvem, sugestão :
+( [Heroku](https://www.heroku.com/), [App Engine](https://cloud.google.com/appengine?utm_source=google&utm_medium=cpc&utm_campaign=latam-BR-all-pt-dr-SKWS-all-all-trial-e-dr-1011454-LUAC0008679&utm_content=text-ad-none-any-DEV_c-CRE_429626774325-ADGP_Hybrid%20%7C%20SKWS%20-%20EXA%20%7C%20Txt%20~%20Compute_App-Engine-KWID_43700040369789872-kwd-359982465286&utm_term=KW_app%20engine-ST_App%20Engine& ...
+- API [ADinfo](https://github.com/DP6/penguin-adinfo#instala%C3%A7%C3%A3o) já hospedada;
```
- Mesmo ponto anterior, padronizar para "Adinfo"
- Existe um motivo para o link estar com âncora?

WernerAssis replied:
o Link ancora manda para a parte de instalação do Adinfo API, como a interface depende da API achei interessante colocar.

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+- Ambiente de hospedagem de aplicações, em nuvem, sugestão :
+( [Heroku](https://www.heroku.com/), [App Engine](https://cloud.google.com/appengine?utm_source=google&utm_medium=cpc&utm_campaign=latam-BR-all-pt-dr-SKWS-all-all-trial-e-dr-1011454-LUAC0008679&utm_content=text-ad-none-any-DEV_c-CRE_429626774325-ADGP_Hybrid%20%7C%20SKWS%20-%20EXA%20%7C%20Txt%20~%20Compute_App-Engine-KWID_43700040369789872-kwd-359982465286&utm_term=KW_app%20engine-ST_App%20Engine& ...
+- API [ADinfo](https://github.com/DP6/penguin-adinfo#instala%C3%A7%C3%A3o) já hospedada;
+- Node a partir da versão 16;
```
Node versão 16 ou superior

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
-For detailed explanation on how things work, consult the [docs for vue-loader](http://vuejs.github.io/vue-loader).
\ No newline at end of file
+## Hospedagem
+A interface do ADinfo, é baseada em *Vue.js*, *Node.js* e *Html & Css*, atualmente esta sendo hospedada no *App Engine*, tendo as configurações de ambiente feitas por *App.yaml*. 	
```
Não mencionar que está hospedado no App Engine

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+## Hospedagem
+A interface do ADinfo, é baseada em *Vue.js*, *Node.js* e *Html & Css*, atualmente esta sendo hospedada no *App Engine*, tendo as configurações de ambiente feitas por *App.yaml*. 	
+
+Também contamos com *Babelrc* para transpilar códigos e manter a compatibilidade com navegadores.
```
Quem transpila o código é o Vue.js, não precisa especificar isso

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+Também contamos com *Babelrc* para transpilar códigos e manter a compatibilidade com navegadores.
+
+## Google App Engine
+Recomendamos fortemente o **Google App Engine**  como ambiente de hospedagem por ser uma opção com escabilidade autómatica, banco de dados e toda infraestrutura que apenas o Google pode oferecer.
```
- Retirar o "fortemente"
- O App Engine não possui banco de dados

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+## Google App Engine
+Recomendamos fortemente o **Google App Engine**  como ambiente de hospedagem por ser uma opção com escabilidade autómatica, banco de dados e toda infraestrutura que apenas o Google pode oferecer.
+
+Além disso ele permite que a aplicação seja executada em host local, e quando feita alterações, ele atualiza automaticamente em tempo real sem necessidade de reimplementar.
```
Retirar esse trecho, o appengine não faz isso

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+
+    https://github.com/DP6/adinfo-interface.git
+
+ **Após a instalação, será necessário apenas iniciar a interface executando os seguintes comandos :**
```
Tirar o "apenas"

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+
+ **Após a instalação, será necessário apenas iniciar a interface executando os seguintes comandos :**
+
+*Instala as dependências para execução local:*
```
Instalar

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+
+    npm install
+
+ *Executa a aplicação em ambiente local(http://localhost:8080/):*
```
- Executar
- Retirar os parenteses e o conteudo dele

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+ *Executa a aplicação em ambiente local(http://localhost:8080/):*
+ 
+
+    nmp start
```
npm

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+
+## 1. Controle de acessos ao Adinfo
+
+### **1.1 Tipos de usuários**
```
Essa informação é da API, não? Não corresponde à interface

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+2.  Preencha os campos em aberto contendo o e-mail do novo usuário e a senha;
+    
+3.  Ao clicar em criar, você receberá em resposta a senha de acesso. Com ela o novo usuário poderá utilizar a interface do Adinfo com acesso de Operação.
+4. Logo abaixo na sub aba de Gerenciamento de Usuários também é possível ativar e desativar quais usuários terão acesso a plataforma de um modo bem eficiente.
```
- Incluir a vírgula em "Logo abaixo, na sub aba"
- Retirar o final "de um modo bem eficiente"

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+
+![](https://lh4.googleusercontent.com/63IHzxnijB35AAeFEVIf_IIy-JWaod0WR9C8ra7klCww9Vw1SuOEKok50yJxDNo-Siki8lTpSrKIXWSlTjwe77avvap1nL-rCUze9iu4NxKAZaJdpp19-tcVHsZfivh9vO0nACfvDJNlJWPoSWG-nGs)
+1. Para acessar o template configurado, selecione a opção Parametrizar no menu lateral.
+2. Em seguida a opção Template (1). 
```
Acho que pode tirar as partes que estão "Em seguida" já que os steps já estão numerados

WernerAssis replied:
pronto✔️

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+1. Para acessar o template configurado, selecione a opção Parametrizar no menu lateral.
+2. Em seguida a opção Template (1). 
+3. Em seguida, clique em Consultar template (2).
+4. Após a conferência dos campos, realize o download do template em um dos dois formatos disponíveis, CSV ou XLSX (3).
```
CSV ou Planilha Excel (XLSX)

WernerAssis replied:
pronto✔️

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+3. Em seguida, clique em Consultar template (2).
+4. Após a conferência dos campos, realize o download do template em um dos dois formatos disponíveis, CSV ou XLSX (3).
+
+*Após o Download, as linhas devem ser preenchidas com as respectivas informações, necessárias para a geração do arquivo de parametrização.*
```
Retirar a vírgula.
"Informações necessárias"

WernerAssis replied:
pronto✔️

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+
+6. Selecione então a ferramenta de Analytics já predefinida.
+
+7. E logo em seguida a mídia desejada.
```
Retirar o "E"

WernerAssis replied:
pronto✔️

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,115 @@
+
+8.  Com todos os campos selecionados, clique na opção parametrizar.
+    
+*Após a parametrização, o Adinfo apresentará uma prévia das informações configuradas para revisão. Para consultar o resultado, selecione a opção Download Parametrização.*
```
Destacar ou colocar aspas em "Download Parametrização"

WernerAssis replied:
pronto✔️

WernerAssis committed:
docs: primeiras correções em relação a validação.

WernerAssis committed:
docs: atualização da documentação.

WernerAssis committed:
docs: atualização da documentação.

WernerAssis committed:
docs: correções da documentação.

Commit:
docs: Correção do testo de App Engine e remoção da sessão dos usuários.

Conforme solicitado, alterei toda a indicação da sessão do app engine para um texto que fizesse sentido e retirei o tópico que explica as divisões de usuários já que corresponde a API.

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+<div  align="center">  <img  src="https://raw.githubusercontent.com/DP6/templates-centro-de-inovacoes/main/public/images/centro_de_inovacao_dp6.png"  height="100px"  />  </div>
 
-## Build Setup
+**Adinfo-interface** é a interface do [penguin Adinfo](https://github.com/DP6/penguin-adinfo) ferramenta que foi desenvolvida para facilitar o controle e padronização do uso de parametrização e nomenclatura de mídia digital. Os principais componentes no uso da aplicação são a configuração, um JSON contendo quais campos são aceitos na taxonomia de mídia, um CSV contendo a lista de URLs e os campos ...
```
Colocar dois pontos depois de "Os principais componentes são"

WernerAssis replied:
ok✔️

LucasTonetto commented on README.md:
````diff
@@ -1,18 +1,134 @@
-# build for production with minification
-npm run build
-```
+A interface foi idealizada para **simplificar** e **otimizar** a parametrização de links de forma altamente customizável, sendo uma ferramenta descomplicada de se utilizar e que irá centralizar de **forma segura** as informações mais relevantes do seu negócio.
````
Retirar esse trecho. Parece uma propaganda da interface. Tente manter uma linguagem mais técnica e explicativa

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+- Ambiente de hospedagem de aplicações sugestão:
+
+( [Heroku](https://www.heroku.com/),  [Google App Engine](https://cloud.google.com/appengine?utm_source=google&utm_medium=cpc&utm_campaign=latam-BR-all-pt-dr-SKWS-all-all-trial-e-dr-1011454-LUAC0008679&utm_content=text-ad-none-any-DEV_c-CRE_429626774325-ADGP_Hybrid%20%7C%20SKWS%20-%20EXA%20%7C%20Txt%20~%20Compute_App-Engine-KWID_43700040369789872-kwd-359982465286&utm_term=KW_app%20engine-ST_App%2 ...
```
Deixar na mesma linha da linha 15 e dar somente um espaço entre as opções

WernerAssis replied:
ok✔️

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
\ No newline at end of file
+# Requisitos para utilização
+
+- Ambiente de hospedagem de aplicações sugestão:
```
A sugestão é apenas o AppEngine, vale citar as demais como "outras", mas não como sugestão

WernerAssis replied:
check

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+- API [Adinfo](https://github.com/DP6/penguin-adinfo#instala%C3%A7%C3%A3o) já hospedada;
+
+- Node a partir da versão 16;
```
- Node versão 16 ou superior

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+- Node a partir da versão 16;
+
+- Vue.js.
```
- Vue.js é um pré-requisito pro desenvolvimento da interface ou pra rodar tbm?

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+## Hospedagem
+
+A interface do Adinfo, é baseada em *Vue.js*, *Node.js* e *Html & Css*, atualmente esta sendo hospedada no *App Engine*, tendo as configurações de ambiente feitas por *App.yaml*.
```
"A interface do Adinfo foi criada utilizando Vue.js, e hospedada no serviço App Engine do Google Cloud Platform. Todas as configurações de deploy da aplicação podem ser encontradas no arquivo App.yaml

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+A interface do Adinfo, é baseada em *Vue.js*, *Node.js* e *Html & Css*, atualmente esta sendo hospedada no *App Engine*, tendo as configurações de ambiente feitas por *App.yaml*.
+
+Também contamos com *Babelrc* para transpilar códigos e manter a compatibilidade com navegadores.
```
Retirar. O Vue usa o Babel por baixo dos panos, não precisamos citar, já que não o usamos diretamente

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+## Google App Engine
+
+Recomendamos fortemente o **Google App Engine** como ambiente de hospedagem por ser uma opção que requer o mínimo de configurações possíveis, com um serviço de plataforma completo e muita segurança do ambiente, o App Engine se mostra muito eficaz e fácil de utilizar.
```
Substituir esse trecho por: Para deploy da aplicação é recomendável o uso do serviço App Engine do Google Cloud Platform.

Tente evitar palavras como "recomendamos fortemente", não há algo que se recomenda fracamente. Geralmente só se usa "recomendar fortemente" pra dar mais ênfase, como essa é uma documentação técnica, não há necessidade disso.
Também evitar trechos que fazem propaganda sobre o Google Cloud, como "completo" "seguro" "simples" "eficaz", o intuito é só recomendar um ambiente de execução, a documentação é técnica, não há necessidade de expor qualidades do ambiente recomendado. O App Engine é recomendado simplesmente porque foi desenvolvido nele e conseguimos garantir a execução neste ambiente

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+Recomendamos fortemente o **Google App Engine** como ambiente de hospedagem por ser uma opção que requer o mínimo de configurações possíveis, com um serviço de plataforma completo e muita segurança do ambiente, o App Engine se mostra muito eficaz e fácil de utilizar.
+
+Além destes pontos, basta apenas enviar o projeto para o App Engine que tudo deve funcionar e qualquer atualização será feita em tempo real sem a necessidade de reimplementação. Consequentemente, não há necessidade de se preocupar em qual plataforma de hardware o serviço está rodando, tanto quanto a versão do sistema operacional e muito menos a configuração de todo o ambiente utilizado.
```
Retirar esse trecho.
Em geral, em documentações técnicas as explicações de deploy não são feitas em texto, apenas um passo a passo de como fazer o deploy é o suficiente

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+Além destes pontos, basta apenas enviar o projeto para o App Engine que tudo deve funcionar e qualquer atualização será feita em tempo real sem a necessidade de reimplementação. Consequentemente, não há necessidade de se preocupar em qual plataforma de hardware o serviço está rodando, tanto quanto a versão do sistema operacional e muito menos a configuração de todo o ambiente utilizado.
+
+  **Requisitos Google App Engine:**
```
Retirar esse tópico de Requisitos Google App Engine.
Apenas faça uma lista de pré-requisitos e coloque o App Engine na lista

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+``https://github.com/DP6/adinfo-interface.git``
+
+- altere o link de direcionamento da interface para API (caminho:  src > main.js);
```
quando estiver indicando diretórios, utilizar / em vez de >
src/main.js

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+- altere o link de direcionamento da interface para API (caminho:  src > main.js);
+- ``Vue.prototype.$apiRoute = 'Link da API'.``
+  
+**Após a instalação, será necessário apenas iniciar a interface executando os seguintes comandos :**
```
Retirar o negrito

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+  
+**Após a instalação, será necessário apenas iniciar a interface executando os seguintes comandos :**
+
+*Instala as dependências para execução local:*
```
Instalação

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+``npm install``
+
+*Executa a aplicação em ambiente local(<http://localhost:8080/>):*
```
Execução

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+# Overview da Interface
+
+## 1. Como adicionar novos usuários ao Adinfo
```
no Adinfo

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+![](https://lh4.googleusercontent.com/d5WKB29ZulGuVLJ4K4jixKZltp9bvFqT8WwYLvcBLDFvpxkhDnh4cif3n3YnOVEfMOf2DzeXwjYOPLvT4HDy3qM3pKq4gK78VN2Qhy8_Yydcq0fe2PiKGOfWhiVP_za0IiuLMtlZOcKpqCKfjybljSc)
+
+1. Através de um usuário com permissão de Adimn, visite a aba de Usuários;
```
Com um usuário com permissão de Admin ou Owner, clique na aba de Usuários

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+![](https://lh4.googleusercontent.com/d5WKB29ZulGuVLJ4K4jixKZltp9bvFqT8WwYLvcBLDFvpxkhDnh4cif3n3YnOVEfMOf2DzeXwjYOPLvT4HDy3qM3pKq4gK78VN2Qhy8_Yydcq0fe2PiKGOfWhiVP_za0IiuLMtlZOcKpqCKfjybljSc)
+
+1. Através de um usuário com permissão de Adimn, visite a aba de Usuários;
+
```
Acreidto que o step 2 seria clicar em "Registrar Usuário"

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+1. Através de um usuário com permissão de Adimn, visite a aba de Usuários;
+
+2. apos registrar usuários preencha os campos em aberto contendo o e-mail do novo usuário e a senha;
```
Preencha os campos em aberto

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+3. Ao clicar em criar, você receberá em resposta a senha de acesso. Com ela o novo usuário poderá utilizar a interface do Adinfo com acesso de Operação.
+
+4. Logo abaixo na sub aba de Gerenciamento de Usuários também é possível ativar e desativar quais usuários terão acesso a plataforma de um modo bem eficiente.
```
Logo abaixo, na sub aba de Gerenciamento de Usuários, é possível ativar e desativar o acesso de usuários.

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+2. apos registrar usuários preencha os campos em aberto contendo o e-mail do novo usuário e a senha;
+
+3. Ao clicar em criar, você receberá em resposta a senha de acesso. Com ela o novo usuário poderá utilizar a interface do Adinfo com acesso de Operação.
```
Todos itens da lista terminam com ;
Com exceção do último que termina com .

WernerAssis replied:
ok✔️

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+### 2.1 **Download do arquivo de preenchimento**
+
+O primeiro passo do processo é a consulta e download do arquivo de preenchimento ou template. O template possui todas as informações definidas pelo Administrador como necessárias nos elementos de nomenclatura e URLs, assim como suas opções de preenchimento e dependências, quando relevantes.
```
URL

WernerAssis replied:
ok✔️

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+![](https://lh4.googleusercontent.com/63IHzxnijB35AAeFEVIf_IIy-JWaod0WR9C8ra7klCww9Vw1SuOEKok50yJxDNo-Siki8lTpSrKIXWSlTjwe77avvap1nL-rCUze9iu4NxKAZaJdpp19-tcVHsZfivh9vO0nACfvDJNlJWPoSWG-nGs)
+
+1. Para acessar o template configurado, selecione a opção Parametrizar no menu lateral.
```
Selecione a opção Parametrizar no menu lateral;

WernerAssis replied:
ok✔️

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+1. Para acessar o template configurado, selecione a opção Parametrizar no menu lateral.
+
+2. selecione a opção Template (1).
```
Todos itens começam com letra maiúscula e terminam com ;
O último item termina com .

WernerAssis replied:
ok✔️

LucasTonetto commented on README.md:
```diff
@@ -1,18 +1,134 @@
+
+6. Selecione então a ferramenta de Analytics já predefinida.
+
+7. em seguida a mídia desejada.
```
Começar com letra maiúscula
Mesma observação sobre ; e . das demais listas

WernerAssis replied:
ok✔️

WernerAssis committed:
docs: atualização da doc.

WernerAssis committed:
docs: correções de escrita.

WernerAssis committed:
docs: atualização e correções.

aqui contemplo os pontos informados na PR.

LucasTonetto commented on README.md:
```diff
@@ -11,20 +11,19 @@
 
 ## Google App Engine
 
 Recomendamos fortemente o **Google App Engine** como ambiente de hospedagem por ser uma opção com escabilidade autómatica, banco de dados e toda infraestrutura que apenas o Google pode oferecer.
```
Retirar essa parte

LucasTonetto commented on README.md:
```diff
@@ -11,20 +11,19 @@
 
 Recomendamos fortemente o **Google App Engine** como ambiente de hospedagem por ser uma opção com escabilidade autómatica, banco de dados e toda infraestrutura que apenas o Google pode oferecer.
 
 Além disso ele permite que a aplicação seja executada em host local, e quando feita alterações, ele atualiza automaticamente em tempo real sem necessidade de reimplementar.
```
Essa também

LucasTonetto commented on README.md:
```diff
@@ -11,20 +11,19 @@
 
+- Google App Engine;
 - Google Cloud SDK;
 após instalação e configuração execute no terminal:
```
Após

LucasTonetto commented on README.md:
```diff
@@ -35,16 +34,16 @@ após instalação e configuração execute no terminal:
 - ``Vue.prototype.$apiRoute = 'Link da API'.``
   
-**Após a instalação, será necessário apenas iniciar a interface executando os seguintes comandos :**
+Após a instalação, será necessário apenas iniciar a interface executando os seguintes comandos :
```
tirar o espaço antes dos :

LucasTonetto commented on README.md:
```diff
@@ -64,9 +63,9 @@ No Adinfo, temos dois tipos dois tipos de usuário:
+1. Com um usuário com permissão de Admin ou Owner, clique na aba de Usuários;
 
-2. apos registrar usuários preencha os campos em aberto contendo o e-mail do novo usuário e a senha;
+2. Clique em registrar usuário, preencha os campos em aberto;
```
2. Clique em registrar usuário e preencha os campos em aberto;

WernerAssis committed:
docs: correções doc.

WernerAssis committed:
docs: correções ortograficas.

WernerAssis committed:
docs: correções ortograficas.
